src/BrowserKitDriver.php

```diff
@@ -413,7 +413,24 @@ public function getValue($xpath)
      */
     public function setValue($xpath, $value)
     {
-        $this->getFormField($xpath)->setValue($value);
+        $formField = $this->getFormField($xpath);
+
+        try {
```

Comment:
It's not possible for DomCrawler to select option by text, but now it possible for BrowserKit.
This improvement add possibility write more readable behat tests, based on what you see, not on html attributes.